src/views/RecIM/views/Admin/index.jsx

```diff
@@ -83,6 +110,33 @@ const Admin = () => {
     setLoading(false);
   }, [user?.IsAdmin]);
 
+  const handleAdminMenuOpen = (e) => {
+    setAdminMenuAnchorEl(e.currentTarget);
+  };
+
+  const handleAdminMenuClose = () => {
+    setAdminMenuAnchorEl(null);
+  };
+
+  const generateAdminReport = () => {
+    let fStartDate = selectedDateIn.toISOString();
+    let fEndDate = selectedDateOut.toISOString();
+    getRecIMReport(fStartDate, fEndDate).then((value) => handleOpenRecimReport(value));
+  };
+
+  const handleOpenRecimReport = (report) => {
+    console.log('report: ');
+    console.log(report);
```

Comment:
Do you still need these logs?

Reply:
I'm leaving them in while I'm still working on this, just to be able to view the raw data.  I'll be sure to remove them before making this a final PR.

Reply:
You can always leave your PR as a draft. Otherwise, it implies that you are intending on merging this as-is. That can be misleading.

Reply:
Yep, I intentionally made this a PR for review, this part of the feature is ready, I was planning on making a separate issue/PR for adding the print functionality, since that ended up being a bigger project than expected.  But, this PR is ready to be merged as-is as far as I'm concerned.

Reply:
Correct, I believe I addressed all your comments, and went through to get rid of all my testing and debug lines/cleanup :+1: